workflow/rules/ref.smk

```diff
@@ -59,9 +59,9 @@ rule get_known_variants:
         "v2.5.0/bio/reference/ensembl-variation"
 
 
-rule get_annotation_gz:
+rule get_annotation:
     output:
-        "resources/annotation.gtf.gz",
+        "resources/annotation.gtf",
```

Comment:
I would not do this here. Am I right that this is only needed for the scripts that postprocess this? In the scripts, you should be able to just read the file as gzip with python: https://docs.python.org/3/library/gzip.html

Reply:
This is indeed not needed for this PR but for the fusion calling PR where the star aligner also requires the annotations. As star does not support gzip-files I decided to already switch to a plain gtf file in this PR as I would just require to overhaul the python script in the other PR.